src/pages/edit_resource/ResourceForm.tsx

```diff
@@ -144,8 +154,9 @@ const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) =
           {(formikProps: FormikProps<FormikValues>) => (
             <StyledForm>
               <ScrollToContentButton contentRef={contentRef} text={t('skip_to_form_content')} />
+              <div tabIndex={0} ref={beforeResourceFormNavigationRef} />
```

Comment:
valgte å scrolle til over stepperen. Det føltes mest naturlig. Alternativt kunne man gått til toppen av siden.

Reply:
Tror du har en bug. Hvis du skrur opp størrelsen på teksten på 400% og kun benytter navigering med keyboard, så vil du ikke klare å legge inn tittel og beskrivelse på ressursen.

Reply:
(så stort at "rediger ressurs" tittelen tar opp hele skjermen.

Reply:
Samme med opphavere. Når man bruker tab for å komme seg videre fra stepperen (uten å hoppe over stepperen) så havner man midt i opphaver feltet.

Reply:
Det virker som om tab-index=-1 fikset den biffen også. Gjerne sjekk så jeg ikke har misforstått. Jeg mener det funker bra hos meg  nå iallefall.